### PR TITLE
Player edition during game

### PR DESCRIPTION
**Pull request by valentinchevalier**
Add the possibility to edit any player score, faults and elimination during a game.
To do so, you have to do a long click on a player, and a modal appear :)

### CODE REVIEW

File: src/stores/game.js [hunk 1]
```diff
@@ -200,9 +220,9 @@ export default {
             }, { root: true });
           }
           break;
-        case rulesTypes.ZAP_RESET:
-          // TODO handle this case
-          break;
+        // case rulesTypes.ZAP_RESET:
```
Comment (tbille) [inline src/stores/game.js]: Do you still wanna keep this code?

File: src/i18n/en.json [hunk 2]
```diff
@@ -9,11 +9,13 @@
...
     "start": "Start game",
-    "max-players": "Maximum number of players reached"
+    "max-players": "Maximum number of players reached",
+    "shuffle": "Shuffle players"
   },
   "game": {
     "scores": {
-      "title": "Scores"
+      "title": "Scores",
+      "eliminated": "Eliminateds"
```
Comment (tbille) [inline src/i18n/en.json]: `Eliminated`

File: src/i18n/en.json [hunk 3]
```diff
@@ -54,5 +56,26 @@
...
+  "modal": {
+    "validate": "ok",
+    "sanctions": {
+      "zap": {
+        "title": "!!! Zap !!!",
+        "content": " | <strong>{players}</strong> go back to {score} points | <strong>{players}</strong> go back to {score} points"
+      },
+      "eliminated": {
+        "title": "Booooh !",
+        "content": "<strong>{name}</strong> is eliminate"
```
Comment (tbille) [inline src/i18n/en.json]: `is eliminated`

File: src/i18n/en.json [hunk 4]
```diff
@@ -54,5 +56,26 @@
...
+      }
+    }
+  },
+  "player-edition": {
+    "title": "<strong>{name}</strong> update",
```
Comment (tbille) [inline src/i18n/en.json]: `updated`

File: src/i18n/en.json [hunk 5]
```diff
@@ -54,5 +56,26 @@
...
+    "fields": {
+      "fault": "Faults : ",
+      "score": "Score : ",
+      "eliminated": "eliminate"
```
Comment (tbille) [inline src/i18n/en.json]: I don't know the context, might be good, might not :stuck_out_tongue:

File: src/i18n/fr.json [hunk 6]
```diff
@@ -67,5 +69,13 @@
         "content": "<strong>{name}</strong> est éliminé"
       }
     }
+  },
+  "player-edition": {
+    "title": "Modifiation de <strong>{name}</strong>",
```
Comment (tbille) [inline src/i18n/fr.json]: `ModifiCation`

**Comment by tbille** [comment]
@valentinchevalier In the modal I can put negative numbers.

File: src/i18n/en.json (hunk 2 above)
Comment (ClementChaumel) [reply src/i18n/en.json]: Or `Eliminated players`

**Pull request merged**